pre-commit.nix

```diff
@@ -43,6 +43,24 @@
           settings = {
             ormolu.cabalDefaultExtensions = true;
             statix.ignore = [ "**spago-packages.nix" ];
+            typos = {
+              config = ''
+                [default.extend-words]
```

Comment:
Please put this in typos config file? There a reason not to?

Reply:
Honestly, I stole this from @szg251 in https://github.com/mlabs-haskell/lambda-buffers-for-cardano/pull/22/commits/c46898227e5bcea6cad2ce10e99572192977419b so perhaps he would know -- but I don't see a reason as to why this shouldn't be in the typos config file. Feel free to chime in @szg251 :)

Implemented in https://github.com/mlabs-haskell/lambda-buffers-for-cardano/pull/23/commits/4eb451418ab0f09e100589270672d492497e56be